examples/aws-profile-py/__main__.py

```diff
@@ -25,6 +28,7 @@
 # Create the cluster using the AWS provider and credential opts.
 cluster = eks.Cluster(project_name,
                       provider_credential_opts=kubeconfig_opts,
+                      # TODO(#1475): bootstrap_self_managed_addons=false, # To speed up the test.
```

Comment:
Just a suggestion -- I noticed the coredns addon adds ~900s to the test, and it would be really great to disable!

Reply:
The culprit here is not aws adding the coredns addon, they do that as a self-managed addon (just a helm chart) and it doesn't wait for coredns to be healthy.

We're now using the managed coredns addon by default. If you want to disable it for this test, you can do so by setting `corednsAddonOptions.enabled` to false. I don't think we need it for this test because we're already testing that with others.

Reply:
This was a huge time saver, thank you!